rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -1138,6 +1138,39 @@ void saveNewFederationBtcUTXOs_utxosShouldBeSavedToStorage(FederationStorageInde
 
     }
 
+    @Test
+    void saveOldFederationBtcUTXOs_utxosShouldBeSavedToStorage() {
+
+        // Arrange
+
+        StorageAccessor storageAccessor = new InMemoryStorage();
+        DataWord oldFederationBtcUtxosKey = OLD_FEDERATION_BTC_UTXOS_KEY.getKey();
+
+        storageAccessor.saveToRepository(oldFederationBtcUtxosKey, new byte[] {});
+        FederationStorageProvider federationStorageProvider = new FederationStorageProviderImpl(storageAccessor);
+
+        Address btcAddress = BitcoinTestUtils.createP2PKHAddress(networkParameters, "test");
+
+        List<UTXO> utxos = federationStorageProvider.getOldFederationBtcUTXOs();
+        List<UTXO> extraUtxos = BitcoinTestUtils.createUTXOs(1, btcAddress);
+
+        utxos.addAll(extraUtxos);
+
+        // Act
+
+        federationStorageProvider.save(networkParameters, activations);
+
+        // Assert
+
+        List<UTXO> finalListOfUtxos = storageAccessor.getFromRepository(oldFederationBtcUtxosKey, BridgeSerializationUtils::deserializeUTXOList);
+
+        assertEquals(2, finalListOfUtxos.size());
+        assertEquals(utxos, finalListOfUtxos);
+        // Ensuring `getOldFederationBtcUTXOs` also returns the one saved in the storage.
+        assertEquals(finalListOfUtxos, federationStorageProvider.getOldFederationBtcUTXOs());
```

Comment:
The `final` is the `actual`, right? Here it should be on the right side of the assertion